pkg/bpmn_engine/variable_scope/local_scope.go

```diff
@@ -0,0 +1,49 @@
+package variable_scope
+
+type LocalScope struct {
```

Comment:
the LocalScope implementation seems redundant to me.
Why not simply use ```localScope := variable_scope.NewScope(nil, nil)```?
So we could simplify the code?